### ISSUE TITLE
No such file or directory: 'requirements.txt'

### ISSUE THREAD

**Issue by tmhammer** [nficano/python-lambda#34]
When following the steps in Getting Started, I see this error when I run `pip install python-lambda`:
```
Collecting python-lambda
  Using cached python-lambda-0.5.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/br/bx8tyrg91fx1pgqwvnwc315w0000gn/T/pip-build-l7DEkV/python-lambda/setup.py", line 16, in <module>
        pip_requirements = [str(r.req) for r in requirements]
      File "/Users/tmh/virtualenvs/pylambda/lib/python2.7/site-packages/pip/req/req_file.py", line 84, in parse_requirements
        filename, comes_from=comes_from, session=session
      File "/Users/tmh/virtualenvs/pylambda/lib/python2.7/site-packages/pip/download.py", line 425, in get_file_content
        'Could not open requirements file: %s' % str(exc)
    pip.exceptions.InstallationError: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/br/bx8tyrg91fx1pgqwvnwc315w0000gn/T/pip-build-l7DEkV/python-lambda/
```

**Comment by nficano** [comment]
sorry about that, pushed a bad release. reverted

**Comment by nielsonsantana** [comment]
I think that this problem is because the version available on pypi (pip) is 0.5.0, and the current is 0.7.0. 
https://pypi.python.org/pypi/python-lambda/

**Comment by jmeekhof** [comment]
Uploaded latest version to pypi.
